src/platform/lumin-runtime/elements/builders/audio-builder.js

```diff
@@ -130,13 +130,22 @@ export class AudioBuilder extends TransformBuilder {
         }
     }
 
+    setSpatialSoundDirection(element, oldProperties, newProperties) {
+        newProperties.spatialSoundDirection
+            .forEach(({ channel, channelDirection }) => {
+                if (channel !== undefined && channelDirection !== undefined) {
+                    element(channel, channelDirection);
```

Comment:
should there be some function call on element here?

Reply:
:+1: Thank you @tonycuadra !